httpserver-rs/src/lib.rs

```diff
@@ -340,6 +351,7 @@ fn convert_response_headers(
                 continue;
             }
         };
+        map.remove(&name);
```

Comment:
Why was this needed?

Reply:
So that if the value is set by the user in the response, that value takes precedence over the one in the linkef. 
@thomastaylor312 can you take a look as well.